Formula/v/veryfasttree.rb

```diff
@@ -0,0 +1,76 @@
+class Veryfasttree < Formula
+  desc "Efficient phylogenetic tree inference for massive taxonomic datasets"
+  homepage "https://github.com/citiususc/veryfasttree"
+  url "https://github.com/citiususc/veryfasttree/archive/refs/tags/v4.0.3.tar.gz"
+  sha256 "770e41773806c65e60ebc77363beccf40e3d339affd97c07eb7b2e1acb51af60"
+  license "GPL-3.0-only"
+  head "https://github.com/citiususc/veryfasttree.git", branch: "master"
+
+  depends_on "cmake" => :build
+  depends_on "libomp"
+
+  def install
+    system "cmake", "-B", "build"
+    system "cmake", "--build", "build"
+    bin.install "build/VeryFastTree" => "VeryFastTree"
```

Comment:
Is there no `cmake --install`?

Reply:
When I run the command 'cmake --install,' it fails because it tries to install (by default) it in '/usr/local/bin/VeryFastTree,' so I use 'bin.install' instead.